CONTRIBUTING.md

```diff
@@ -0,0 +1,45 @@
+# Contributing to leaderboard-backend
+We appreciate your help!
+
+## Before filing an issue
+If you are unsure whether you have found a bug, please consider asking in [our discord](https://discord.gg/TZvfau25Vb) first.
+
+Similarly, if you have a question about a potential feature, [the discord](https://discord.gg/TZvfau25Vb) can be a fantastic resource for first comments.
+
+## Filing issues
+Filing issues is as simple as going to [the issue tracker](https://github.com/speedrun-website/leaderboard-backend/issues), and adding an issue!
+
+## Contributing code
+### Example code contribution flow
+1. Make a fork of this repo.
+1. Name a branch on your fork something descriptive for this change (eg. `UpdateMakefile`).
+1. Commit your changes (Tip! Please read our [Style guide](#style-guide) to help the pull request process go smoothly).
+1. Verify your changes work with `make test`.
```

Comment:
Do we want to add a guideline that we intend for unit tests to be added with new features unless you have a good justification not to?

Reply:
Ha I almost thought that went without saying, but yes let's say that.

Reply:
I usually would think the same, but it does seem like we're going to have a lot of people getting into this as their first big open source project so just airing on the side of being explicit!

Reply:
Added an explicit line 👍